.github/workflows/release_6_docs.yml

```
@@ -33,7 +33,9 @@ jobs:
       run: yarn build
 
     - name: (Release - Docs 6.0) Build Website
-      run: yarn antora ./antora-playbook-staging.yml
+      run: yarn antora ./antora-playbook-release.yml
```

```suggestion
      run: yarn antora ./antora-playbook-prod.yml
```
Just noticed this name is incorrect and this should be the correct one.

---

It could be worth just merging `release/docs-6` to `staging/docs-6` first since https://github.com/tinymce/tinymce-docs/commit/a4e28eb4a23fd60db010d6e26ba8940311fc2372 exists on the release branch but not the staging branch.